### PR TITLE
Fixes of issues from Sonar and other code quality plugins

### PR DESCRIPTION
**Pull request by mprtcz**

### CODE REVIEW

File: src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java [hunk 1]
```diff
@@ -24,17 +24,29 @@
...
     public GridPane playersGridPane;
+    @FXML
     public VBox opponentsVBox;
+    @FXML
     public VBox playerNameVBox;
+    @FXML
     public Label opponentsLabel;
+    @FXML
     public GridPane opponentsGridPane;
+    @FXML
```
Comment (piotr921) [inline src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/client/network/SocketClient.java [hunk 2]
```diff
@@ -14,10 +14,10 @@
...
  */
 public class SocketClient {
-    private final static String HOST_NAME = "localhost";
-  
-    final static int PORT = 6666;
-    private final static Logger logger = Logger.getLogger(AppLogger.class.getName());
+    private static final String HOST_NAME = "localhost";
+
+    static final int PORT = 6666;
+    private static final Logger logger = Logger.getLogger(AppLogger.class.getName());
```
Comment (piotr921) [inline src/main/java/com/spanish_inquisition/battleship/client/network/SocketClient.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java [hunk 3]
```diff
@@ -26,11 +26,12 @@ public static void initializeLogger() {
...
-            logger.log(Level.WARNING, "Could not create file");
-            e.printStackTrace();
+            logger.log(Level.WARNING, "Could not create file", e);
         }
         handler.setFormatter(new SimpleFormatter());
         logger.addHandler(handler);
         logger.setLevel(Level.CONFIG);
     }
+
+    private AppLogger() {}
```
Comment (piotr921) [inline src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java]: 👍

File: src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java [hunk 4]
```diff
@@ -26,11 +26,12 @@ public static void initializeLogger() {
         try {
             handler = new FileHandler("Battleship.log", false);
         } catch (IOException e) {
-            logger.log(Level.WARNING, "Could not create file");
-            e.printStackTrace();
+            logger.log(Level.WARNING, "Could not create file", e);
```
Comment (Ice94) [inline src/main/java/com/spanish_inquisition/battleship/common/AppLogger.java]: :+1:

File: src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java [hunk 5]
```diff
@@ -24,17 +24,29 @@
  */
 public class MainMenuController {
 
+    @FXML
     public VBox centralVBox;
```
Comment (Ice94) [inline src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java]: What is centralVBox?

File: src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java [hunk 6]
```diff
@@ -24,17 +24,29 @@
...
+    @FXML
     public TextField nameTextField;
+    @FXML
     public Button featureButton;
```
Comment (Ice94) [inline src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java]: What is featureButton?

**Pull request merged**